packages/appstore/src/types/props.types.ts

```diff
@@ -1,10 +1,3 @@
-import ConfigStore from 'Stores/config-store';
```

Comment:
why do we need these changes here?

Reply:
Not very sure about this changes, @yashimwong did this for implementing router configuration to app store.